datumaro/components/annotation.py

```diff
@@ -110,6 +110,7 @@ def from_iterable(cls, iterable: Iterable[Union[
 
         Args:
             iterable: This iterable object can be:
+
```

Comment:
Do we need these empty lines?

Reply:
Empty lines are needed for correct display on the site

Reply:
It there a way to clearly understand how it will look like and if we need to insert a newline? I find this logic pretty obscure.

Reply:
This [tutorial](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#lists-and-quote-like-blocks) says:
> Nested lists are possible, but be aware that they must be separated from the parent list items by blank lines